test/test_public_bindings.py

```diff
@@ -261,7 +248,14 @@ def test_correct_module_names(self):
 
         def test_module(modname):
             split_strs = modname.split('.')
-            mod = sys.modules.get(modname)
+            try:
+                if "__main__" in modname:
+                    return
+                mod = importlib.import_module(modname)
+            except Exception:
+                # It is ok to ignore here as we have a test above that ensures
+                # this should never happen
+                return
```

Comment:
If we have a test to ignore, can we just get rid of the except?

If we cannot get rid of it, can we make it more fine-grained (e.g. ImportError?) Otherwise if there is a syntax error in line 252-254, we will erroneously suppress it

Reply:
Not really because we don't know what to skip here and some module import fail with something other than `ImportError`.
But I agree this should be improved to be made more reliable.